Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/NoticestoMarinersControllerTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Web.Controllers;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class NoticesToMarinersControllerTest
+    {
+        private NoticesToMarinersController _controller;
+        private INMDataService _nMService;
+        private ILogger<NoticesToMarinersController> _logger;
+        private IHttpContextAccessor _contextAccessor;
+        private INMDataService _nMDataService;
+
+        public const string CorrelationId = "7b838400-7d73-4a64-982b-f426bddc1296";
+
+        [SetUp]
+        public void Setup()
+        {
+            _nMService = A.Fake<INMDataService>();
+            _logger = A.Fake<ILogger<NoticesToMarinersController>>();
+            _contextAccessor = A.Fake<IHttpContextAccessor>();
+            _nMDataService = A.Fake<INMDataService>();
+            A.CallTo(() => _contextAccessor.HttpContext).Returns(new DefaultHttpContext());
+            _controller = new NoticesToMarinersController(_nMService, _contextAccessor, _logger);
+        }
+
+        [Test]
+        public void WhenIndexIsCalled_ThenShouldReturnsExpectedView()
+        {
+            string expectedView = "~/Views/NoticesToMariners/FilterWeeklyFiles.cshtml";
```

Comment:
`const` where possible please:
```suggestion
            const string expectedView = "~/Views/NoticesToMariners/FilterWeeklyFiles.cshtml";
```